lib/buyingbehavior.rb

```diff
@@ -63,7 +63,7 @@ def buy_cheapest_best_quality_with_prime
   end
 
   def buy_cheapest_best_quality
-    best_quality = $items.map { |item| item['quality'] }.max
+    best_quality = $items.map { |item| item['quality'] }.min
```

Comment:
Wouldn't it make sense to use the finding_best_quality method here as well?

Reply:
I am scared by Ruby code. :D
But that's a good point. Will take a look at it.

Reply:
Both alternatives seem good to me.
You can delete `finding_best_quality` if you are not using it. It's not used anywhere else.